src/Raven.Client/ServerWide/Operations/UpdateUnusedDatabasesOperation.cs

```diff
@@ -0,0 +1,70 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Net.Http;
+using System.Text;
+using Raven.Client.Documents.Conventions;
+using Raven.Client.Documents.Operations;
+using Raven.Client.Documents.Session;
+using Raven.Client.Http;
+using Raven.Client.Json;
+using Raven.Client.Util;
+using Sparrow.Json;
+
+namespace Raven.Client.ServerWide.Operations
+{
+    public class UpdateUnusedDatabasesOperation : IServerOperation
+    {
+        private readonly string _database;
+        private readonly HashSet<string> _unusedDatabaseIds;
+
+        public UpdateUnusedDatabasesOperation(string database, HashSet<string> unusedDatabaseIds)
+        {
+            if (string.IsNullOrEmpty(database))
+                throw new ArgumentException(database);
+
+            _database = database;
+            _unusedDatabaseIds = unusedDatabaseIds;
+        }
+
+        public RavenCommand GetCommand(DocumentConventions conventions, JsonOperationContext context)
+        {
+
+            return new UpdateUnusedDatabasesCommand(_database, _unusedDatabaseIds);
+        }
+
+        private class UpdateUnusedDatabasesCommand : RavenCommand, IRaftCommand
+        {
+            private readonly string _database;
+            private readonly Parameters _parameters;
+
+            public UpdateUnusedDatabasesCommand(string database, HashSet<string> unusedDatabaseIds)
+            {
+                _database = database;
+                _parameters = new Parameters
+                {
+                    DatabaseIds = unusedDatabaseIds
+                };
+            }
+
+            public override HttpRequestMessage CreateRequest(JsonOperationContext ctx, ServerNode node, out string url)
+            {
+                url = $"{node.Url}/admin/unused-databases?database={_database}";
+
+                return new HttpRequestMessage
+                {
+                    Method = HttpMethod.Post,
+                    Content = new BlittableJsonContent(stream =>
+                    {
+                        ctx.Write(stream, EntityToBlittable.ConvertCommandToBlittable(_parameters, ctx));
+                    })
+                };
+            }
+
+            public string RaftUniqueRequestId => RaftIdGenerator.NewId();
+        }
+        public class Parameters
```

Comment:
can be private, but need to put in inside the command

Reply:
Can't be private. It is used also in `JsonDeserializationServer`

Reply:
Then let's move it inside UpdateUnusedDatabasesCommand  and make it internal